lib/govobject/types/trigger.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable */
```

Comment:
I think we don't need it since we are writing new code?

Reply:
@shumkov linter check will fail otherwise, as the code is ES5, but linter checks against modern standard